### PR TITLE
Add Macs with max macOS being Catalina

### PR DESCRIPTION
**Pull request by MichalMMac**
Attempt to add models with macOS 10.15.6 as a maximum supported macOS version.

I hope I understood the data file structure.

### CODE REVIEW

File: supported_os_data.yml [hunk 1]
```diff
@@ -101,24 +101,28 @@ highest:
     '21': 10.7.5
     '31': 10.11.6
     '41': 10.13.6
-    '61': '0'
+    '61': 10.15.6
```
Comment (tuxudo) [inline supported_os_data.yml:104]: Macmini6,2 is the last for 10.15.6 support

File: supported_os_data.yml [hunk 2]
```diff
@@ -101,24 +101,28 @@ highest:
...
+    '71': '0'
   MacBookPro:
     '11': 10.6.8
     '21': 10.7.5
     '31': 10.11.6
     '61': 10.13.6
-    '91': '0'
+    '91': 10.15.6
```
Comment (tuxudo) [inline supported_os_data.yml:111]: MacBookPro10,2 is the last for 10.15.6

File: supported_os_data.yml [hunk 3]
```diff
@@ -101,24 +101,28 @@ highest:
...
+    '111': '0'
   MacBookAir:
     '11': 10.7.5
     '21': 10.11.6
     '31': 10.13.6
-    '51': '0'
+    '51': 10.15.6
```
Comment (tuxudo) [inline supported_os_data.yml:117]: MacBookAir5,2 is the last for 10.15.6

File: supported_os_data.yml [hunk 4]
```diff
@@ -101,24 +101,28 @@ highest:
...
+    '61': '0'
   iMac:
     '41': 10.6.8
     '51': 10.7.5
     '71': 10.11.6
     '101': 10.13.6
-    '131': '0'
+    '131': 10.15.6
```
Comment (tuxudo) [inline supported_os_data.yml:124]: iMac14,3 is the last for 10.15.6

**Comment by tuxudo** [comment]
current_os at the top should also be changed to `11.0.0`

**Comment by MichalMMac** [comment]
I clearly don't understand the data format here. I thought key represents the model number of the first Mac for which is the specific relase the highest possible. 

It would be probably safer to close this.

**Comment by tuxudo** [comment]
The key is the last model that can support the OS version. It's kinda backwards how it's done

**Comment by tuxudo** [comment]
My mistake, it is a really confusing thing and you are correct.

**Pull request closed without merging**